src/lib/select/select.scss

```diff
@@ -100,5 +105,7 @@ $mat-select-placeholder-arrow-space: 2 * ($mat-select-arrow-size + $mat-select-a
     // Remove the transition to prevent the placeholder
     // from overlapping when the label comes back down.
     transition: none;
+    // Prevents the '...' from showing on the parent element.
+    display: block;
```

Comment:
Wouldn't the parent element just not have `overflow: ellipses`?

Reply:
The `overflow: ellipses` on the parent is also what gives the value its `...` so we need to do it this way even though its a little strange